lib/checkPullRequestReview.js

```diff
@@ -381,9 +382,70 @@ const handleResponseToReview = async (context) => {
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ * @param {string[]} assignees
+ * @param {string} comment
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+
+//This is a function to check if there are more than 24 Hours with the Pr
+const isMaxReviewPeriodExceeded = async (review) => {
+  let timeOfReviewSubmission = review.submitted_at;
+  timeOfReviewSubmission = new Date(timeOfReviewSubmission);
+  let currentTime = new Date();
+  let diff = Math.abs(timeOfReviewSubmission - currentTime);
+  return diff > MAX_REVIEW_PERIOD;
+};
+
+/**
+ * This function handles the case for reviewers who have not reviewed PR
+ * for more than 24 hrs.
+ * Then this function pings those reviewers who satisfy this situtation.
+ *
+ * @param {import('probot').Context} context
+ */
+const checkForReviewersWithPendingReview = async (context) => {
+  const allOpenPullRequests = await getAllOpenPullRequests(context);
+  await allOpenPullRequests.map(
+    async (pullRequest) => {
+      const { data: allReviews } = await context.github.pulls.listReviews(
+        context.repo({
+          pull_number: pullRequest.number,
+        })
+      );
+
+      const reviewersToPing = [];
+      allReviews.map((review) => {
+        const isReviewPeriodExceeded = isMaxReviewPeriodExceeded(review);
+        // If Max Period Exceeded then We need to ping that reviewer.
+        if (isReviewPeriodExceeded) {
+          const reviewer = review.user.login;
+          if (!reviewersToPing.includes(reviewer)) {
+            reviewersToPing.push(reviewer);
+          }
+        }
+      });
+
+      if (reviewersToPing.length !== 0) {
+        const message = 'Hi' + reviewersToPing.map((reviewer) => {
+          return ' @' + reviewer.login + ' ';
+        }) + ', Its been more than 24 hours, ' +
+         'since the review was requested. Please review this. Thanks!';
```

Comment:
We are not taking the time when the review was requested but when the review was submitted right?

Reply:
Sorry for that, Will change it in the latest commit Sir. Thanks.

Reply:
No, this is not correct, What I meant is that the comment was correct, but the code is wrong.

Reply:
Sir, Please could you be more specific about where the code is wrong and how it could be corrected. So I can start working on this. Thanks.

Reply:
So, in your code you look at all the reviews and if the review is older than 24 hours you notify the reviewer, but that doesn't make sense. We want to notify the reviewer if the REQUEST for review is older than 24 hours.

Reply:
It does make sense, Thanks for pointing it out. Will perform the changes in the latest commits Sir.

Reply:
@vojtechjelinek  and @jameesjohn I am thinking of the following approach, Could you please confirm whether this is alright.
- Check in the list of requested reviewers.
- Then mapping on to that list and noticing their last reviews. If their review was done before 24 HRS, then ping them .

Does this approach look alright to you sir. Please let me know so i can proceed in this PR.Thanks.

Reply:
That is still the same solution right? The solution should be 'if the review was requested more than 24 hours ago and the reviewer didn't do anything on the PR notify them'.

Reply:
@jameesjohn Sir Could you help me out with how can I find the time for the requested review, I didn't find anything related to it. It would be a big help if you could guide me on this one. Thanks.

Reply:
@vojtechjelinek, we may not be able to get exactly when the review got requested from the pull request data. 
But we can assume that all the currently assigned need to review the pull request, and we can check if they have performed the review or check if they have commented on the pull request. 
Then if they haven't performed any of those, we notify them.

What do you think?

Reply:
@jameesjohn Sir According to me, this approach is the same as currently being implemented by me. Am I right?. Thanks.

Reply:
Yes, I think so.

Reply:
@vojtechjelinek, Sir Could you please reply to the same conversation. Thanks.